packages/codegen-ui-golden-files/lib/react/forms/datastore-update-form-with-has-many-relationship/SchoolUpdateForm.jsx

```diff
@@ -252,6 +253,9 @@ export default function SchoolUpdateForm(props) {
           const studentsToUnLink = [];
           const studentsSet = new Set();
           const linkedStudentsSet = new Set();
+          if (!canUnlinkStudents && studentsToUnLink.length > 0) {
+            throw Error(`${original.id} cannot be unlinked from School because schoolID is a required field.`);
```

Comment:
Not quite sure what "original" is?
Shouldn't the ID coming from studentsToUnlink[0]?
Also, if it's easy enough, suggesting "Student" in the front - 
Student ${studentsToUnlink[0].id} cannot be unlinked from School because schoolID is a required field.`

Reply:
ahh this one is on the golden file. I will move it over to the correct place. But the snapshot should have it in correct place